packages/sommelier.rb

```diff
@@ -2,106 +2,256 @@
 
 class Sommelier < Package
   description 'Sommelier works by redirecting X11 and Wayland programs to the built-in ChromeOS wayland server.'
-  homepage 'https://chromium.googlesource.com/chromiumos/containers/sommelier'
-  version '1382ce084cc407'
+  homepage 'https://chromium.googlesource.com/chromiumos/platform2/+/HEAD/vm_tools/sommelier/'
+  version '20201025'
```

Comment:
Do we have a newer version available since this has been sitting in the queue for so long?

Reply:
No new updates for the past 5 weeks:

![image](https://user-images.githubusercontent.com/1096701/100104008-9b617f80-2e33-11eb-88fb-3c767c815176.png)